3ds/libsidplay/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# Maintainer:  Dave Murphy <davem@devkitpro.org>
+# Contributor: badda71 <me@badda.de>
+
+pkgname=3ds-libsidplay
+pkgver=1.36.60
+pkgrel=1.0
```

Comment:
package release should be a counter: `1`

Reply:
done